forge.config.js

```diff
@@ -1,8 +1,9 @@
 /* tslint:disable */
 
-const path = require('path');
-const package = require('./package.json');
+const path = require('path')
+const packackeJson = require('./package.json')
 
+const { arch, version } = packackeJson;
```

Comment:
How does one get `arch` from `package.json`?

Reply:
It doesn't, this was a bad copy/paste job from another internal app of mine 😅